src/plugins/intel_gpu/src/kernel_selector/cl_kernels/convolution_gpu_ref.cl

```diff
@@ -62,7 +63,7 @@ KERNEL(kernel_name)(
 
     for (uint k = 0; k < FILTER_IFM_NUM; ++k)
     {
-#if INPUT0_SIZE > 4
+#if INPUT0_DIMS > 4
```

Comment:
Maybe make a equal names for the ranks in static and dynamic cases? Now it's `_DIMS`, but for not to change in all the following agnostic kernels `_SIZE` -> `_DIMS` I suggest leaving one old name `_SIZE` and return the name back in the already changed kernels.

Reply:
looks like `_SIZE` and `_DIMS` macros always have the same value `t.GetDims().size()` regardless tensor type, `_SIZE` macro is just not defined for dynamic tensors. So I'd say that this is a good change, but I think we also need to remove `_SIZE` macro at all from jitter as at the first glance it's used in conv ref kernel only.

Reply:
I've deleted creating _SIZE from jitter because this is only used on conv ref kernel.